toolkit/offchain/src/init_governance/transaction.rs

```diff
@@ -0,0 +1,137 @@
+use crate::{csl::*, plutus_script::PlutusScript};
+use cardano_serialization_lib::*;
+use ogmios_client::types::OgmiosUtxo;
+use partner_chains_plutus_data::version_oracle::VersionOracleDatum;
+use sidechain_domain::MainchainAddressHash;
+
+// Script ID of the governance script in the script cache.
+// TODO: Use a proper value of raw_scripts::ScripId once we upgrade to a version that has it.
+const SCRIPT_ID: u32 = 32;
+
+#[allow(dead_code)]
+pub(crate) fn init_governance_transaction(
+	multi_sig_policy: &[u8],
+	version_oracle_validator: &[u8],
+	version_oracle_policy: &[u8],
+	governance_authority: MainchainAddressHash,
+	tx_context: &TransactionContext,
+	genesis_utxo: OgmiosUtxo,
+	ex_units: ExUnits,
+) -> anyhow::Result<Transaction> {
+	let multi_sig_policy =
+		PlutusScript::from_wrapped_cbor(multi_sig_policy, LanguageKind::PlutusV2)?
+			.apply_uplc_data(multisig_governance_policy_configuration(governance_authority))?;
```

Comment:
Maybe you could introduce a bit of wrapping here for the repeated `PlutusScript::from_wrapped_cbor` calls. Something like:

```Rust
fn create_plutus_script(
    policy_bytes: &[u8],
    uplc_data: uplc::PlutusData,
) -> anyhow::Result<PlutusScript> {
    PlutusScript::from_wrapped_cbor(policy_bytes, LanguageKind::PlutusV2)?
        .apply_uplc_data(uplc_data)
}
```

so then you get
```Rust
    let multi_sig_policy = create_plutus_script(
        multi_sig_policy,
        multisig_governance_policy_configuration(governance_authority),
    )?;
```

Reply:
Eh, I don't think that would be useful. Each script can be parametrized with different values so we'd need to have a separate function per each script.

Reply:
Okay, fair enough